src/routes.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
General comment on client side: Would it make sense to have a main component that renders an unauthorized page/redirects to login if this.props.token/loggedInUser is not present, otherwise show the app? Just a general idea for now, haven't thought too much about this yet.